### PR TITLE
Flux 0.68 changelog

### PR DESCRIPTION
**Pull request by sanderson**

### CODE REVIEW

File: content/v2.0/reference/release-notes/flux.md [hunk 1]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
...
+## Breaking Changes
+- Change signature of `group()` function.
+
+## Features
+- Add [`fieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys/) and
+  [`measurementFieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/measurementfieldkeys/)
+  to v1 package.
+- Add a context to `plantest.RuleTestCase`.
+- Add Snowflake support to SQL package.
+- Add `experimental.chain()` function.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/flux.md:33]: link to experimental.chain function?

File: content/v2.0/reference/release-notes/flux.md [hunk 2]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
...
+- Add location information to type error messages.
+- Add all Linux cross-compilation tools to release Docker image.
+- Support remote `buckets()` and `v1.databases()` calls.
+- Add support for static linking.
+- Add `influxdb` source.
+- Add support for `pkg-config`.
+- Transform semantic nodes back to AST nodes.
+- Handle multi-file packages.
+- Make `Eval()` and `EvalAST()` use libflux for parsing and analysis.
+- Add `lookuptype` function for stdlib builtins.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/flux.md:49]: `lookuptype()`?

Comment (kelseiv) [reply content/v2.0/reference/release-notes/flux.md:49]: link to function?

File: content/v2.0/reference/release-notes/flux.md [hunk 3]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
...
+  [`measurementFieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/measurementfieldkeys/)
+  to v1 package.
+- Add a context to `plantest.RuleTestCase`.
+- Add Snowflake support to SQL package.
+- Add `experimental.chain()` function.
+- Add semantic nodes for bad statement and bad expression.
+- Add [`findColumn()`](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findcolumn/)
+  and [`findRecord()`](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findrecord/) functions.
+- Return `false` if `contains()` is called with an empty set.
+- Add small performance optimizations.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/flux.md:38]: Add/Refine performance

File: content/v2.0/reference/release-notes/flux.md [hunk 4]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
...
 
+## v0.68.0 [2020-05-28]
+This version of Flux introduces an updated type inference system that improves
+performance, error messaging, and usability of the
+[Flux Language Server Protocol (LSP)](https://github.com/influxdata/flux-lsp).
+
+## Breaking Changes
+- Change signature of `group()` function.
+
+## Features
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/flux.md:27]: being the list is so long, maybe we can add grouping for scanability:

**New functions**
- Add [`fieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys/) and
  [`measurementFieldKeys()`](/v2.0/reference/flux/stdlib/influxdb-v1/measurementfieldkeys/)
  to v1 package.
- Add [`findColumn()`](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findcolumn/)
  and [`findRecord()`](/v2.0/reference/flux/stdlib/built-in/transformations/stream-table/findrecord/) functions.
- Add `experimental.chain()` function.
- Add `lookuptype` function for stdlib builtins.

**Function updates**
- Make `Eval()` and `EvalAST()` use libflux for parsing and analysis.
- Return `false` if `contains()` is called with an empty set.
- Support remote `buckets()` and `v1.databases()` calls.

**Package updates**
- Handle multi-file packages.
- Add support for `pkg-config`.
- Add Snowflake support to SQL package.

**Node updates**
- Transform semantic nodes back to AST nodes.
- Add semantic nodes for bad statement and bad expression.


**Other features**
- Add a context to `plantest.RuleTestCase`.
- Add performance optimizations.
- Add a dynamically linked Valgrind test.
- Add location information to type error messages.
- Add all Linux cross-compilation tools to release Docker image.
- Add support for static linking.
- Add `influxdb` source.

File: content/v2.0/reference/release-notes/flux.md [hunk 5]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
...
+- Validate HTTP redirects against private IPs.
+- Do not disclose DNS info in HTTP.
+- Fix concurrent map write in `filter()` transformation.
+- Copy all fields of `WindowProcedureSpec` in `Copy()`.
+- Run `go generate` on libflux when `go generate` is run on stdlib.
+- `map()` panics when overwriting group column.
+- Support execution contexts in the REPL.
+- Apply substitution fully when compiling lambda.
+- Planner rewrite rules take a context.
+- Functions do not panic on null values.
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/flux.md:67]: does line 67 describe the fix or the bug?

File: content/v2.0/reference/release-notes/flux.md [hunk 6]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
...
+- Support remote `buckets()` and `v1.databases()` calls.
+- Add support for static linking.
+- Add `influxdb` source.
+- Add support for `pkg-config`.
+- Transform semantic nodes back to AST nodes.
+- Handle multi-file packages.
+- Make `Eval()` and `EvalAST()` use libflux for parsing and analysis.
+- Add `lookuptype` function for stdlib builtins.
+
+## Bug Fixes
```
Comment (kelseiv) [inline content/v2.0/reference/release-notes/flux.md:51]: Same for grouping bug fixes list--function updates, test updates...?

File: content/v2.0/reference/release-notes/flux.md [hunk 7]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
 InfluxDB until the next InfluxDB v2.0 release._
 {{% /note %}}
 
+## v0.68.0 [2020-05-28]
+This version of Flux introduces an updated type inference system that improves
+performance, error messaging, and usability of the
+[Flux Language Server Protocol (LSP)](https://github.com/influxdata/flux-lsp).
+
+## Breaking Changes
+- Change signature of `group()` function.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/reference/release-notes/flux.md:25]: The signature isn't user facing. It's only on the backend and doesn't actually change the way `group()` works. It's listed as a breaking change, but I'm not sure why.

File: content/v2.0/reference/release-notes/flux.md (hunk 4 above)
Comment (sanderson) [reply content/v2.0/reference/release-notes/flux.md:27]: I thought about this, but it breaks convention from all our other release notes.

File: content/v2.0/reference/release-notes/flux.md (hunk 1 above)
Comment (sanderson) [reply content/v2.0/reference/release-notes/flux.md:33]: Docs for the function don't exist yet. Still waiting on feedback from the engineers.

File: content/v2.0/reference/release-notes/flux.md [hunk 8]
```diff
@@ -16,6 +16,121 @@ Though newer versions of Flux may be available, they will not be included with
...
+- Reenable Clippy linter rule match single binding.
+- Fix bug in object equal method.
+- Add builtin formatting.
+- Implement TimeBounds for `influxdb.fromRemote`.
+- Inject the URL validator into `NewDefaultClient`.
+- Fix race condition in the `filter()` function.
+- Validate HTTP redirects against private IPs.
+- Do not disclose DNS info in HTTP.
+- Fix concurrent map write in `filter()` transformation.
+- Copy all fields of `WindowProcedureSpec` in `Copy()`.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/reference/release-notes/flux.md:61]: These are Go functions, so the camel-case is appropriate. There are letter-casing conventions Go.

**Pull request merged**